PRASCore.jl/src/Results/Shortfall.jl

```diff
@@ -201,17 +205,17 @@ function getindex(x::ShortfallResult)
     return sum(x.shortfall_mean), x.shortfall_std
 end
 
-function getindex(x::ShortfallResult, r::AbstractString)
+function getindex(x::ShortfallResult, r::AbstractString)  
```

Comment:
Looks like there's some extra trailing whitespace at the end of the these function lines